src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -1782,7 +1820,8 @@ import './Menu.css'
 | `.pf-m-drilldown` | `.pf-c-menu` | Modifies the menu so that all nested `.pf-c-menu` elements "drill down". |
 | `.pf-m-current-path` | `.pf-c-menu.pf-m-drilldown .pf-c-menu__list-item` | Modifies the menu list item for current path state. |
 | `.pf-m-drilled-in` | `.pf-c-menu.pf-m-drilldown, .pf-c-menu.pf-m-drilldown .pf-c-menu` | Modifies the menu list for drilled in state. |
-| `.pf-m-static` | `.pf-c-menu.pf-m-drilldown .pf-c-menu` | Modifies the menu list for drilled static state. |
+| `.pf-m-static` | `.pf-c-menu.pf-m-drilldown .pf-c-menu` | Modifies the menu list for drilled static state. | 
+| `.pf-m-danger` | `pf-c-menu__item-text` | Modifies the menu item text to indicate danger styling. |
```

Comment:
updated this to match the load/loading modifiers.

```suggestion
| `.pf-m-danger` | `.pf-c-menu__list-item` | Modifies a list item for danger styles. |
```

Reply:
Thanks again for these reviews. Definitely helping me along.